Mathlib/Topology/StoneCech.lean

```diff
@@ -238,6 +244,30 @@ instance stoneCechSetoid : Setoid (Ultrafilter α) where
       @fun _ _ _ xy yz γ _ _ _ f hf => (xy γ f hf).trans (yz γ f hf)⟩
 #align stone_cech_setoid stoneCechSetoid
 
+-- Do we really not have this ??
+lemma Continuous.closedEmbedding_of_compact_t2 {α β : Type*} [TopologicalSpace α]
+    [TopologicalSpace β] [CompactSpace α] [T2Space β] {f : α → β} (hf : Continuous f)
+    (hf' : Function.Injective f) : ClosedEmbedding f :=
+  closedEmbedding_of_continuous_injective_closed hf hf'
+    fun _ hK ↦ hK.isCompact.image hf |>.isClosed
```

Comment:
We have `Continuous.isClosedMap` and `closedEmbedding_of_continuous_injective_closed`.

Reply:
Ah, thanks :facepalm: 
I was thinking that `compact` had to appear in the name.